plugins/CuraEngineBackend/CuraEngineBackend.py

```diff
@@ -1088,11 +1094,17 @@ def disableTimer(self) -> None:
             self._change_timer.timeout.disconnect(self.slice)
 
     def _onPreferencesChanged(self, preference: str) -> None:
-        if preference != "general/auto_slice":
+        if preference != "general/auto_slice" and preference != "info/send_engine_crash":
             return
-        auto_slice = self.determineAutoSlicing()
-        if auto_slice:
-            self._change_timer.start()
+        if preference == "general/auto_slice":
+            auto_slice = self.determineAutoSlicing()
+            if auto_slice:
+                self._change_timer.start()
+        elif preference == "info/send_engine_crash":
+            if CuraApplication.getInstance().getPreferences().getValue("info/send_engine_crash"):
+                os.environ["use_sentry"] = str(1)
+            else:
+                os.environ["use_sentry"] = str(0)
```

Comment:
Think this notation is cleaner for this instance
```suggestion
            os.environ["use_sentry"] = "1" if CuraApplication.getInstance().getPreferences().getValue("info/send_engine_crash") else "0"
```